linux/setup_01.md

```diff
@@ -14,7 +14,7 @@ sudo apt-get install terminator
 
 ## Oh-my-zsh
 
-Jest to menadżer paczek zsh, który dostarcza nam stylowanie składni, aliasy czy wsparcie dla Git
+Jest to framework, zawierający wiele udogodnień dla naszej konsoli
```

Comment:
Chodziło mi bardziej o przeskok 

oh-my-zsh - tak to jest framework (ale dlaczego zsh a nie bash, i czym jest bash :-)